mdal/frmts/mdal_binary_dat.cpp

```diff
@@ -78,6 +78,26 @@ static bool readIStat( std::ifstream &in, int sflg, char *flag )
   return false;
 }
 
+static MDAL::Duration convertTimeData( double time, const std::string &originalTimeDataUnit )
```

Comment:
I am wondering if these conversion functions cannot be part of the new file with datatime in namespace like DataTimeUtils {} or so.. not sure if they are repeating or different for each format?

Reply:
I agree if formats treat units is same way, but if the don't, I think particular treatment have to be in the file of the format.

Reply:
Finally, I have implemented a unique function in MDAL::utils that parses the unit string to retrieve the unit.